src/Mod/TechDraw/App/ProjectionAlgos.cpp

```diff
@@ -0,0 +1,290 @@
+/***************************************************************************
+ *   Copyright (c) Jürgen Riegel          (juergen.riegel@web.de) 2002     *

```

Comment:
Fix heading formatting (should look like [this](https://github.com/FreeCAD/FreeCAD/pull/5457/files#diff-06fc5002222d6a7ea53e9ddb9a565cf709e7ea9ec62841246412d436f05a6edbR2))

Reply:
Why is this important?  The copyright notice has been formatted this way for 20 years.  

Not trying to be obnoxious, just don't understand.